addons/binding/org.openhab.binding.homie/src/test/resources/simulateHomieMQTT.js

```diff
@@ -0,0 +1,75 @@
+var basetopic="homie/";
```

Comment:
This goes into a separate test bundle, Kai mentioned that when I had my tests included in the Yamaha binding PR.